publish.sh

```diff
@@ -2,61 +2,102 @@
 
 set -ex
 
-KUBEVIRTCI_TAG=$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)
-export KUBEVIRTCI_TAG
+export KUBEVIRTCI_TAG=$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)
+
+function detect_cri() {
+    if podman ps >/dev/null 2>&1; then echo podman; elif docker ps >/dev/null 2>&1; then echo docker; fi
```

Comment:
This is what we are doing on all repos already
even on `cluster-up/cluster/kind/check-cluster-up.sh`

Note that IIRC we don't need the socket here, the socket is needed only for gocli provision or alike,
because gocli provision command uses golang docker lib which need the socket

Reply:
Yes you are right - the publishing of kubevirtci providers is usually handled by the automation anyway - should be fine.